sys/linux/auto.txt

```diff
@@ -0,0 +1,375 @@
+open_by_handle_at$auto(mountdirfd intptr, handle intptr, flags intptr) (auto)
+open_tree$auto(dfd intptr, filename intptr, flags intptr) (auto)
+openat$auto(dfd intptr, filename intptr, flags intptr, mode intptr) (auto)
+openat2$auto(dfd intptr, filename intptr, how intptr, usize intptr) (auto)
+pause$auto() (auto)
+perf_event_open$auto(attr_uptr intptr, pid intptr, cpu intptr, group_fd intptr, flags intptr) (auto)
+personality$auto(personality intptr) (auto)
+pidfd_getfd$auto(pidfd intptr, fd intptr, flags intptr) (auto)
+pidfd_open$auto(pid intptr, flags intptr) (auto)
+pidfd_send_signal$auto(pidfd intptr, sig intptr, info intptr, flags intptr) (auto)
+pipe$auto(fildes intptr) (auto)
+pipe2$auto(fildes intptr, flags intptr) (auto)
+pivot_root$auto(new_root intptr, put_old intptr) (auto)
+pkey_alloc$auto(flags intptr, init_val intptr) (auto)
+pkey_free$auto(pkey intptr) (auto)
+pkey_mprotect$auto(start intptr, len intptr, prot intptr, pkey intptr) (auto)
+poll$auto(ufds intptr, nfds intptr, timeout_msecs intptr) (auto)
+ppoll$auto(ufds intptr, nfds intptr, tsp intptr, sigmask intptr, sigsetsize intptr) (auto)
+prctl$auto(option intptr, arg2 intptr, arg3 intptr, arg4 intptr, arg5 intptr) (auto)
+pread64$auto(fd intptr, buf intptr, count intptr, pos intptr) (auto)
+preadv$auto(fd intptr, vec intptr, vlen intptr, pos_l intptr, pos_h intptr) (auto)
+preadv2$auto(fd intptr, vec intptr, vlen intptr, pos_l intptr, pos_h intptr, flags intptr) (auto)
+prlimit64$auto(pid intptr, rsrc intptr, new_rlim intptr, old_rlim intptr) (auto)
+process_madvise$auto(pidfd intptr, vec intptr, vlen intptr, behavior intptr, flags intptr) (auto)
+process_mrelease$auto(pidfd intptr, flags intptr) (auto)
+process_vm_readv$auto(pid intptr, lvec intptr, liovcnt intptr, rvec intptr, riovcnt intptr, flags intptr) (auto)
+process_vm_writev$auto(pid intptr, lvec intptr, liovcnt intptr, rvec intptr, riovcnt intptr, flags intptr) (auto)
+pselect6$auto(n intptr, inp intptr, outp intptr, exp intptr, tsp intptr, sig intptr) (auto)
+ptrace$auto(request intptr, pid intptr, addr intptr, data intptr) (auto)
+pwrite64$auto(fd intptr, buf intptr, count intptr, pos intptr) (auto)
+pwritev$auto(fd intptr, vec intptr, vlen intptr, pos_l intptr, pos_h intptr) (auto)
+pwritev2$auto(fd intptr, vec intptr, vlen intptr, pos_l intptr, pos_h intptr, flags intptr) (auto)
+quotactl$auto(cmd intptr, special intptr, id intptr, addr intptr) (auto)
+quotactl_fd$auto(fd intptr, cmd intptr, id intptr, addr intptr) (auto)
+read$auto(fd intptr, buf intptr, count intptr) (auto)
+readahead$auto(fd intptr, off_lo intptr, off_hi intptr, count intptr) (auto)
+readdir$auto(fd intptr, dirent intptr, count intptr) (auto)
+readlink$auto(path intptr, buf intptr, bufsiz intptr) (auto)
+readlinkat$auto(dfd intptr, pathname intptr, buf intptr, bufsiz intptr) (auto)
+readv$auto(fd intptr, vec intptr, vlen intptr) (auto)
+reboot$auto(magic1 intptr, magic2 intptr, cmd intptr, arg intptr) (auto)
```

Comment:
At least this one is problematic. We need to somehow remove it.
I assume if you try to run syz-manager with this syscall enabled, it will reboot all the time.
I think the easiest way to remove it for now is to explicitly special-case it in run.go.

Reply:
Done! Along with utimesat which is not defined for all archs.